### DISCUSSION TITLE
Do turning lanes count as lanes?

### DISCUSSION THREAD

**Discussion by endolith** [streetcomplete/StreetComplete#4651] [Feedback]
It says bicycle and dedicated parking lanes don't count, but turning lanes do, right?

**Comment by matkoniecz** [comment]
yes, turning lanes count

**Reply by endolith** [reply]
How do we add turning lanes that are in the middle of the road and can be used by either direction?

**Reply by matkoniecz** [reply]
I think there is a special option in "can't say"
